tests/test_element.py

```diff
@@ -42,3 +42,21 @@ def test_element_type() -> None:
 
     assert_type(div()["a"], Element)
     assert isinstance(div()["a"], Element)
+
+
+@pytest.mark.parametrize(
+    ("element", "expected"),
+    [(del_, "<del></del>"), (class_, "<class></class>"), (break_, "<break></break>")],
+)
+def test_keyword_named_elements(element: Element, expected: str) -> None:
+    actual = str(element)
+    assert actual == expected
+
+
+@pytest.mark.parametrize(
+    ("element", "expected"),
+    [(imaginary, "<imaginary></imaginary>"), (imaginary_, "<imaginary-></imaginary->")],
```

Comment:
Should underscore always be replaced with a hyphen like this?

Reply:
I cannot thing of much other useful things to do. Another option would be to raise an error for trailing underscores for non keywords but... it feels to harsh too. 😆